src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -243,12 +246,12 @@ class FacetSidebar extends React.Component {
                     >
                       <input
                         id={`${field}-${valueLabel}`}
-                        aria-labelledby={`${field} ${valueLabel}`}
+                        aria-labelledby={`${field}-${valueLabel}`}
```

Comment:
It still doesn't work as it should. When you click on the text, the checkbox should get checked. I think it's because the `label` and the `input` elements have the same id. So the label has
```
id={`${field}-${valueLabel}`}
htmlFor={`${field}-${valueLabel}`}
```
which probably confuses it. Try making the label's id `id={`${field}-${valueLabel}-label`}` and then updating the `input`'s aria-labelledby to the same value `${field}-${valueLabel}-label`.